pocketimdb/movies/models.py

```diff
@@ -23,3 +23,15 @@ class MovieLike(models.Model):
 
     class Meta:
         unique_together = ('movie', 'user',)
+
+class WatchList(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE, related_name='user_watch_list')
+    movie_watch = models.ManyToManyField(Movie, through='MovieWatch')
+
+class MovieWatch(models.Model):
+    movie = models.ForeignKey(Movie, on_delete=models.CASCADE)
+    watch_list = models.ForeignKey(WatchList, on_delete=models.CASCADE)
+    watched = models.BooleanField(default=False)
```

Comment:
mislim da nemamo potrebe za ovim, mozemo napraviti samo model `MovieWachlist` koji ce da sadrzi user-a, movie i doatno polje `is_watched`, posmatraj to kao na neku pivot tabelu izmedju movie i user-a stim sto imaju dodatno polje i semantiku Wathlist-e

Reply:
Pretpostavio sam da je ovo suvisno, mislim jasno mi je zbog cega i prvobitno dam i hteo da napravim samo jednu tabelu. Ali me je povukao nacin na koji je task opisan da bespotrebno zakomplikujem model, a dodatno sam razmisljao da bi jednog dana u teoriji watch lista mogla da bude prosirena nekim poljima pa da bi bilo lepo da to izolujem.
Ali svakako da to ovde i nema neke potrebe za tim.